### PR TITLE
Add pytz to setup.py

### PR DESCRIPTION
**Pull request by mattmb**
I guess this used to be captured as a dependency of something...

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -56,6 +56,7 @@
         'pyramid == 1.7',
         'pysensu-yelp >= 0.2.2',
         'pytimeparse >= 1.1.0',
+        'pytz == 2014.10',
```
Comment (nhandler) [inline setup.py]: Do we really need this specific version? I imagine any recent version would be sufficient. Can you switch this to a `>=` relationship? Timezone databases change all the time, so supporting more recent versions of the module would be preferred.

**Comment by nhandler** [comment]
Fix and Ship!

**Comment by solarkennedy** [comment]
ship

File: setup.py (hunk 1 above)
Comment (EvanKrall) [inline setup.py]: or at least do something from 2016, like 2016.6.1

**Comment by mattmb** [comment]
I duped this from requirements.txt which is why it's pinned. I agree that there's probably no need, I'll unpin and then ship this if the tests pass.

**Pull request merged**